tests/main.test.ts

```diff
@@ -208,12 +209,12 @@ describe("User start/stop", () => {
       await userStartStop(context);
     } catch (error) {
       if (error instanceof Error) {
-        expect(error.message).toEqual("Too many assigned issues, you have reached your max limit of 3 issues.");
+        expect(error.message).toEqual("Too many assigned issues, you have reached your max limit of 2 issues.");
```

Comment:
Ideally this should be picked up from the configuration or the test so the string is dynamic maybe, so if we change it the tests doesn't fail? Forget about this if it's too much work.

Reply:
@gentlementlegen I think it's best that @jordan-ae also does as I was advised by whilefoo in my PR. My implementing of the tests in this way above was poor and should not be used.

@jordan-ae you don't have to change any other tests just the new ones that you are adding/refactoring make sure they use the below format.

```ts
const {config} = context
const adminLimit = config.maxTasks[0].limit

await expect(userStartStop(context)).rejects.toThrow(`Too many assigned issues, you have reached your max limit of ${adminLimit} issues.`);
```


[my pr for reference](https://github.com/ubiquibot/command-start-stop/pull/19/commits/5d487fb3d14cb369c9f47cde5fc3fde9715168a6#diff-12cd00a89af03f1fc6f74460b886289151edce062c60319b0c58fb8465558ed6)